{{cookiecutter.project_shortname}}/Pipfile

```diff
@@ -7,11 +7,12 @@ verify_ssl = true
 check-manifest = ">=0.25"
 
 [packages]
-invenio-app-ils = {extras = ["postgresql", "elasticsearch7", "lorem"], version = "~=1.0.0rc1"}
+invenio-app-ils = {extras = ["postgresql", "opensearch2", "lorem"], version = "~=4.0.0"}
 setuptools = ">=57.0.0,<58.0.0"
 uwsgi = ">=2.0"
 uwsgitop = ">=0.11"
 uwsgi-tools = ">=1.1.1"
+invenio-cli = ">=1.4.0"
```

Comment:
it needs to be installed before intalling from pipfile (globally in your system) so it is not needed here